.github/deploy-to-lambda.yml

```diff
@@ -0,0 +1,33 @@
+name: deploy to lambda
```

Comment:
Make one more of this workflow file for a staging lambda!
and obviously, you would need to create a staging lambda, staging API Gateway etc

The two files will be like `deploy-to-lambda-staging` and `deploy-to-lambda-prod`. And the staging github action would be triggered on a pull request (just like pre-commit gets triggered)
And production github action would be triggered when a branch is merged to release.

For reference, you can take a look at the github actions files in plio frontend or backend

Reply:
So ideally, when you committed this code, it should've deployed to a staging lambda which we can test instantly. I can't see anything deployed right now

Reply:
Also, did you look into setting up the API keys? Please post an update for whatever comment that is present! Resolved means done and if not resolved then please give a return comment - otherwise we'll keep playing ping pong :p

Reply:
Why is the pre-commit not being triggered on a PR?
Did you put it in the workflow file?

Ideal behaviour (just as it happens in plio) -- whenever you make a PR , two things happen
- the code of this PR gets deployed on a staging lambda
- the pre commit will run telling us if all linting is fine or not

And, whenever you merge a branch into release, 
- the release branch gets deployed on prod lambda

Reply:
> 

I'm confused, why a separate staging? Since it's a service, won't the lambda be the exact same?